appveyor.yml

```diff
@@ -20,4 +20,4 @@ install:
 - make init
 
 test_script:
-- tox
+- tox
```

Comment:
Why are we running tox here? We have the matrix above which is doing what tox would do anyways.

Reply:
No idea, this was here before.
The only change here is removing the last newline which I'm not sure is a good idea or necessary, I'm going to revert it.